README.rst

```diff
@@ -92,6 +92,15 @@ To run selected single/multiple tests under longrunning package::
 Here, ``-Dlong.test`` is used to specify multiple comma separated tests.
 ``-Dlong.running.namespace`` is used to specify namespace name for all long running tests. If not specified, 'Default' namespace will be used.
 
+CDAP perf-tests
+------------------
+```
+mvn clean test -P perf-tests -Dtest=DBBaseTest -Ddatabase.connectionString="jdbc:mysql://localhost:3306/demo" \
```

Comment:
You don't mention that this manual is only for running single test. It looks like all perf tests should start the same way

Reply:
removed from this example, and placed it in DBSpecificTest example